app/views/content_items/news_article.html.erb

```diff
@@ -11,6 +10,7 @@
 </div>
 
 <%= render 'shared/publisher_metadata_with_logo' %>
+<%= render 'components/notice', @content_item.withdrawal_notice_component  %>
```

Comment:
@miaallers similarly this looks like it will render notices at full width not 2/3rds width. Is this right?